packages/integrations/netlify/package.json

```diff
@@ -50,5 +50,8 @@
     "cheerio": "^1.0.0-rc.11",
     "mocha": "^9.2.2",
     "vite": "^4.0.3"
+  },
+  "astro": {
+    "external": true
```

Comment:
😞 Not sure how I feel about this... Maybe it would be better to hardcode the list if they're all first-party?

Reply:
The problem is that they're not necessarily first party, any SSR adapter can need it

Reply:
We talked about this and said we weren't going to document it for now. If the concern is about having an API to maintain; I had the same concern.

Reply:
Cool, think we're all on the same page. Not ideal but it makes sense. Resolving my comment.